src/module/rules/language.mjs

```
@@ -1,39 +1,36 @@
 
-    //Deleting Languages
-    delete CONFIG.DND5E.languages.standard.children.giant
-    delete CONFIG.DND5E.languages.standard.children.gnomish 
-    delete CONFIG.DND5E.languages.standard.children.orc
-    delete CONFIG.DND5E.languages.exotic
-    delete CONFIG.DND5E.languages.druidic;
-    delete CONFIG.DND5E.languages.cant;
+    // Deleting Languages
+    const languagesToDelete = ["giant", "gnomish", "orc", "druidic", "cant"];
+    languagesToDelete.forEach(lang => delete CONFIG.DND5E.languages.standard.children[lang]);
+    delete CONFIG.DND5E.languages.exotic;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)